README.rst

```diff
@@ -91,7 +91,7 @@ If you run ``hansel -h``, you should see the following usage statement:
 
           Subtype Salmonella enterica genomes using 33bp k-mer typing schemes.
           Includes schemes for Heidelberg and Enteritidis subtyping.
-          Developed by Genevi�ve Labb�, James Robertson, Peter Kruczkiewicz, Chad R. Laing, Kim Ziebell, Marisa Rankin, Aleisha R. Reimer, Lorelee Tschetter, Gary Van Domselaar, Eduardo N. Taboada, Sadjia Bekal, Kimberley A. MacDonald, Linda Hoang, Linda Chui, Danielle Daignault, Durda Slavic, Frank Pollari, E. Jane Parmley, Philip Mabon, Elissa Giang, Lok Kan Lee, Jonathan Moffat, Joanne MacKinnon, Benjamin M. Hetman, Roger Johnson, John H.E. Nash.
+          Developed by Genevieve Labbe, James Robertson, Peter Kruczkiewicz, Chad R. Laing, Kim Ziebell, Marisa Rankin, Aleisha R. Reimer, Lorelee Tschetter, Gary Van Domselaar, Eduardo N. Taboada, Sadjia Bekal, Kimberley A. MacDonald, Linda Hoang, Linda Chui, Danielle Daignault, Durda Slavic, Frank Pollari, E. Jane Parmley, Philip Mabon, Elissa Giang, Lok Kan Lee, Jonathan Moffat, Joanne MacKinnon, Benjamin M. Hetman, Roger Johnson, John H.E. Nash.
```

Comment:
Unicode not displaying properly for you? Looks fine in master.

Reply:
Shows question marks where the acccent aigu and grave are supposed to be.

Reply:
Yeah looks like it in the diff, but maybe it's alright when rendered to HTML from RST?

Reply:
Perhaps, even using `hansel -h` results in the accents missing :(